### ISSUE TITLE
[semantic-issue] Add semantics for sklearn.metrics mean_squared_error

### ISSUE THREAD

**Issue by FrancescoBernini** [spangea/Pyra#151]
**Description**
Notebook: ztgg-project-1.py

**Minimal running example**
```python
from sklearn.metrics import mean_squared_error
test_accuracy = mean_squared_error(1, 1)
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html#sklearn-metrics-mean-squared-error

**Note (optional)**
Mean squared error regression loss.

Returns:
lossfloat or ndarray of floats
A non-negative floating point value (the best value is 0.0), or an array of floating point values, one for each individual target.

File ".../lyran/src/lyra/semantics/statistical_type_semantics.py", line 66, in relaxed_open_call_policy
    raise Exception(f"Semantics for {stmt} at line  {stmt.pp.line} not yes implemented")
Exception: Semantics for mean_squared_error(CVKFold#y_test_pred, CVKFold#y_test) at line  188 not yes implemented
